Mu2eG4/geom/geom_2021_PhaseI_v03.txt

```diff
@@ -75,7 +75,7 @@ double mu2e.detectorSystemZ0 = 10171.;   // mm  G4BL: (17730-7292=9801 mm)
 #include "Offline/Mu2eG4/geom/tracker_v6.txt"
 
 // Crystal calorimeter
-#include "Offline/Mu2eG4/geom/calorimeter_CsI.txt"
+#include "Offline/Mu2eG4/geom/calorimeter_CsI_v2.txt"
```

Comment:
Until we officially declare MDC2020 a legacy branch later this month, we want to keep backwards compatibility. I think this should be v1.

Reply:
I tested switching to the `calorimeter_CsI_v1` file and rebasing to the head of main and I see a change in the calorimeter origin when accessing the calorimeter geom utils:
```diff
< Calo geom info (Disk = 0): origin = (-3904,0,12036.3) 
---
> Calo geom info (Disk = 0): origin = (-3904,0,12046.3) 
```
With `calorimeter_CsI_v2` I see another origin:
```diff
> Calo geom info (Disk = 0): origin = (-3904,0,12069.3)
```
It looks like this might be the reason the trigger rates have changed. Are these origin shifts expected?

Reply:
Oh, this should be the case. Let me investigate

Reply:
Apparently, I just failed elementary school calculus, 42-33 isn't 19 but 9. I'll fix this later.

Reply:
sounds like arithmetic... Let me know when this is ready to test again.